src/App.jsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import './App.css';
+import { Helmet } from 'react-helmet';
 import logo from './logo.svg';
 
 function App() {
   return (
     <div className="App">
+      <Helmet>
+        <title>Spotties</title>
```

Comment:
We're using `spotties` all lowercased. That might change once we have a logo, but for now, let's keep it all lowercased.